torch/testing/_internal/schema_check_tensor.py

```diff
@@ -30,6 +34,14 @@ def __new__(cls, elem):
         r.elem = elem
         return r
 
+    @staticmethod
+    def reset_cache():
+        SchemaCheckTensor.recorded_ops.clear()
```

Comment:
Any reason not to make the recorded ops per-instance? It seems maybe more intuitive that if you have multiple `SchemaCheckTensors` floating around, that they would only record the ops that they were run on (instead of sharing / modifying a shared list).

Reply:
(just realized I'm basically echo'ing what Nick said :) )